packages/cli/test/unit/commands/list.test.ts

```diff
@@ -75,6 +79,84 @@ describe('list', () => {
       });
       const deployment = useDeployment({ creator: user });
 
+      client.setArgv('-S', user.username);
+      await list(client);
+
+      const output = await readOutputStream(client, 4);
+
+      const { org } = pluckIdentifiersFromDeploymentList(output.split('\n')[0]);
+      const header: string[] = parseSpacedTableRow(output.split('\n')[3]);
+      const data: string[] = parseSpacedTableRow(output.split('\n')[4]);
+      data.shift();
+
+      expect(org).toEqual(user.username);
+      expect(header).toEqual(['Age', 'Deployment URL', 'Status', 'Duration']);
+      expect(data).toEqual([
+        'https://' + deployment.url,
+        stateString(deployment.state || ''),
+        getDeploymentDuration(deployment as unknown as Deployment),
```

Comment:
why is the type casting necessary here? `useDeployment` should return a `Deployment`

Reply:
This is necessary because there are two `Deployment` types: one in `@vercel/client` and one in `cli/src/types`. The one in `cli` is used by `list` and the one in `client` is used by the mock deployments.

This should probably be fixed, but I would imagine that's for a different PR. But happy to do it in this one.

Reply:
Turns out I was stupid and it was actually very easy to fix. Fixed in latest commit